documentation/src/main/asciidoc/advanced-features.asciidoc

```diff
@@ -19,6 +20,22 @@ SearchFactory searchFactory = fullTextSession.getSearchFactory();
 ----
 ====
 
+[[AccessingSearchIntegrator]]
+=== Accessing the SearchIntegrator
```

Comment:
`IndexReader` and `SearchFactory` appear in the same page and none of them use back quotes for formatting. Should it be changed everywhere?